sycl/test-e2e/Plugin/sycl-ls-gpu-default-level-zero.cpp

```diff
@@ -1,4 +1,4 @@
-// REQUIRES: gpu, level-zero
+// REQUIRES: gpu, level_zero
```

Comment:
I prefer dash to underscore as it's easier to type. If there are no strong reasons for using underscore for LIT feaures, I suggest we use dash instead.

Reply:
I have no objections, but that's a question for another PR. Here I'd say both options should be treated as correct.

Reply:
[d76ffef](https://github.com/intel/llvm/pull/16019/commits/d76ffef2998f93f8ced827946a76a21a2f6078fd)

Reply:
> Here I'd say both options should be treated as correct.

Okay, but we must remove the duplication.